pyproject.toml

```diff
@@ -27,7 +27,7 @@ dev-dependencies = [
 [tool.uv.sources]
 
 [project]
-requires-python = ">=3.9"
+requires-python = ">=3.11"
```

Comment:
Can we go with 3.12 or 3.13 instead? 3.11 is now security status.

Reply:
changed to 3.12